charts/prometheus-adapter/templates/psp.yaml

```
@@ -13,6 +13,11 @@ spec:
   {{- if .Values.hostNetwork.enabled }}
   hostNetwork: true
   {{- end }}
+  {{- if .Values.hostNetwork.hostPortsEnabled }}
+  hostPorts:
+    - min: {{ .Values.listenPort }}
+      max: {{ .Values.listenPort }}
+  {{- end }}
```

Do we need this option? Since the container ports can't be customized if you set `hostNetwork=true` it will implicitly use hostPorts for the listen port. Can't we just always set hostPorts to listenPort if hostnetwork is true?
```suggestion
  {{- if .Values.hostNetwork.enabled }}
  hostNetwork: true
  hostPorts:
    - min: {{ .Values.listenPort }}
      max: {{ .Values.listenPort }}
  {{- end }}
```